src/main.rs

```diff
@@ -808,3 +824,34 @@ pub fn get_pkg_name_from_cargo_toml(cargo_toml_file: &str) -> Option<String> {
     let pkg = manifest.package?;
     Some(pkg.name)
 }
+
+pub async fn send_job_to_remote(
+    repo_url: &str,
+    commit_hash: &Option<String>,
+    program_id: &Pubkey,
+    library_name: &Option<String>,
+    bpf_flag: bool,
+) -> anyhow::Result<()> {
+    let client = Client::new();
+
+    // Send the POST request
+    let response = client
+        .post("https://verify.osec.io/verify")
+        .json(&json!({
+                "repository": repo_url,
+                "commit_hash": commit_hash,
+                "program_id": program_id.to_string(),
+                "lib_name": library_name,
+                "bpf_flag": bpf_flag,
```

Comment:
Is this sufficient? shouldn't the full request be sent to the server?

Reply:
I've updated our API with additional optional arguments.